packages/documentation/src/stories/components/table/table.stories.tsx

```diff
@@ -9,10 +10,181 @@ export default {
-  argTypes: {},
+  args: {
+    variant: 'null',
+    caption: 'List of users',
+    captionPlacement: 'bottom',
+    alignment: 'align-top',
+    hover: false,
+  },
+  argTypes: {
+    caption: {
+      name: 'Caption',
+      description: 'A description of the table, very useful for screen reader users.',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Caption',
+      },
+    },
+    captionPlacement: {
+      name: 'Caption Placement',
+      description:
+        'Choose where to place the description of the table. If a visual description is not necessary, it can also be hidden visually and still be of value for screen reader users.',
+      control: {
+        type: 'select',
+        labels: {
+          bottom: 'Bottom (default)',
+          top: 'Top',
+          hidden: 'Visually hidden',
+        },
+      },
+      options: ['bottom', 'top', 'hidden'],
+      table: {
+        category: 'Caption',
+      },
+    },
+    variant: {
+      name: 'Variants',
+      description: 'Stylistic table variants.',
+      control: {
+        type: 'radio',
+        labels: {
+          'null': 'Default',
+          'table-striped': 'Striped',
+          'table-bordered': 'Bordered',
+          'table-borderless': 'Borderless',
+        },
+      },
+      options: ['null', 'table-striped', 'table-bordered', 'table-borderless'],
+      table: {
+        category: 'General',
+      },
+    },
+    alignment: {
+      name: 'Alignment',
+      description:
+        'Vertical alignment options for cell content. Alignment classes can also be set for individual rows or cells.',
+      control: {
+        type: 'select',
+        labels: {
+          'align-top': 'Top (default)',
+          'align-middle': 'Middle',
+          'align-bottom': 'Bottom',
+        },
+      },
+      options: ['align-top', 'align-middle', 'align-bottom'],
+      table: {
+        category: 'General',
+      },
+    },
+    hover: {
+      name: 'Hover',
+      description: 'Enable or disable hover styles.',
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        category: 'General',
+      },
+    },
+  },
 } as Meta;
 
-const Template = (args: Args) => <></>;
+const TableWrapper = (props: ReactProps<{ args: Args; children: ChildNode }>) => (
+  <table
+    className={[
+      'table',
+      props.args.variant,
+      props.args.captionPlacement === 'top' ? 'caption-top' : 'null',
+      props.args.hover ? 'table-hover' : 'null',
+      props.args.alignment,
+    ]
+      .filter(a => a !== 'null')
+      .join(' ')}
+  >
+    <caption className={props.args.captionPlacement === 'hidden' ? 'visually-hidden' : ''}>
+      {props.args.caption}
+    </caption>
+    <thead>
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">First</th>
+        <th scope="col">Last</th>
+        <th scope="col">Handle</th>
+      </tr>
+    </thead>
+
+    <tbody>{props.children}</tbody>
+  </table>
+);
```

Comment:
The output code of react components is not visible in the code output.
You should not use them as helpers components.
Instead you should include the components html direct in the template.

Instead:
- Move the HTML from the TableWrapper component directly in to the first template.
- Add an arg 'content' to the existing args.
- Add the property `table: { disable: true }` to the argTypes definition of this control to remove it from the UI.
- Add the property in the template like it would represent a slot:
```javascript
const Template = (args: Args) => (
  <table
    className={[
      'table',
      args.variant,
      args.captionPlacement === 'top' ? 'caption-top' : 'null',
      args.hover ? 'table-hover' : 'null',
      args.alignment,
    ]
      .filter(a => a !== 'null')
      .join(' ')}
  >
    <caption className={props.args.captionPlacement === 'hidden' ? 'visually-hidden' : ''}>
      {props.args.caption}
    </caption>
    <thead>
      <tr>
        <th scope="col">#</th>
        <th scope="col">First</th>
        <th scope="col">Last</th>
        <th scope="col">Handle</th>
      </tr>
    </thead>

    <tbody>{args.content}</tbody>
  </table>
);
```
- Use this arg as `React.ReactElement[]` per story and move the content from the first template to its value for the default story:
```javascript
export const Default: Story = Template.bind({});
Default.args = {
  content: [
    <tr>
      <th scope="row">1</th>
      <td>Mark</td>
      <td>Otto</td>
      <td>@mdo</td>
    </tr>
    <tr>
      <th scope="row">2</th>
      <td>Jacob</td>
      <td>Thornton</td>
      <td>@fat</td>
    </tr>
    <tr>
      <th scope="row">3</th>
      <td>Larry</td>
      <td>the Bird</td>
      <td>@twitter</td>
    </tr>
  ]
};
``` 
- Do the same for the second story, so the second template should become obsolete.

I did not test the given example code ;)
But I hope you get the Idea.

Reply:
fixed